telemetry-aware-scheduling/deploy/cluster-api/generic/capi.md

```diff
@@ -0,0 +1,166 @@
+# Cluster API deployment - Generic provider
+
+**This guide is meant for local testing/development only, this is not meant for production usage.**
+
+For the deployment using the Docker provider (local testing/development only), please refer to [Cluster API deployment - Generic provider](capi.md).
+
+## Requirements
+
+- A management cluster provisioned in your infrastructure of choice and the relative tooling.
+  See [Cluster API Quickstart](https://cluster-api.sigs.k8s.io/user/quick-start.html).
+- Run Kubernetes v1.22 or greater (tested on Kubernetes v1.25).
+
+## Provision clusters with TAS installed using Cluster API
+
+We will provision a generic cluster with the TAS installed using Cluster API. This was tested using a the GCP provider.
+
+1. Enable the `EXP_CLUSTER_RESOURCE_SET` feature gate:
+
+```bash
+export EXP_CLUSTER_RESOURCE_SET=true
+```
+
+2. In your management cluster, with all your environment variables set to generate cluster definitions, run for example:
+
+```bash
+clusterctl generate cluster capi-quickstart \
+  --kubernetes-version v1.25.0 \
+  --control-plane-machine-count=1 \
+  --worker-machine-count=3 \
+  > capi-quickstart.yaml
+```
+
+If Kind was running correctly, and the Docker provider was initialized with the previous command, the command will return nothing to indicate success.
```

Comment:
I don't think this is needed, seems like a remnant from the Docker/Kind installation

Reply:
This would be something I added after [this comment](https://github.com/intel/platform-aware-scheduling/pull/108#discussion_r1088846896), should it be removed?